pom.xml

```diff
@@ -56,6 +56,14 @@
         <confluent.maven.repo>http://packages.confluent.io/maven/</confluent.maven.repo>
         <apacheds-jdbm1.version>2.0.0-M2</apacheds-jdbm1.version>
         <kafka.connect.maven.plugin.version>0.11.1</kafka.connect.maven.plugin.version>
+        <maven.release.plugin.version>2.5.3</maven.release.plugin.version>
+        <kafka.connect.storage.common.version>10.0.0</kafka.connect.storage.common.version>
+        <confluent-log4j.version>1.2.17-cp2</confluent-log4j.version>
+        <commons.collections.version>3.2.2</commons.collections.version>
+        <libthrift.version>0.13.0</libthrift.version>
+        <log4j-api.version>2.8.2</log4j-api.version>
+        <htrace-core4.version>10.0.0</htrace-core4.version>
```

Comment:
That's also in a different version too probably. Maybe it's worth following CP versioning on branches before 5.5.x